scripts/sanity-sdk/secure-ownable/ownership-transfer-tests.ts

```
@@ -0,0 +1,811 @@
+      // Check for existing pending ownership transfer transactions (like sanity test does)
+      console.log('  🔍 Checking for existing pending ownership transfer transactions...');
+      const pendingTxs = await this.secureOwnable.getPendingTransactions();
+      
+      let txId: bigint | null = null;
+      
+      // Look for ownership transfer transaction in pending transactions
+      if (pendingTxs.length > 0) {
+        for (const pendingTxId of pendingTxs) {
+          try {
+            const tx = await secureOwnableRecovery.getTransaction(pendingTxId);
+            // Check if this is an ownership transfer transaction
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)